core/modules/image/image.field.inc

```diff
@@ -200,6 +204,23 @@ function image_field_instance_settings_form($field, $instance) {
   return $form;
 }
 
+/**
+ * #element_validate handler for image_field_instance_settings_form().
+ */
+function image_field_settings_form_validate($element, &$form_state, $complete_form) {
+  $max_dimensions_x = $form_state['input']['instance']['settings']['max_dimensions']['x'];
+  $max_dimensions_y = $form_state['input']['instance']['settings']['max_dimensions']['y'];
+  $min_dimensions_x = $form_state['input']['instance']['settings']['min_dimensions']['x'];
+  $min_dimensions_y = $form_state['input']['instance']['settings']['min_dimensions']['y'];
+
+  // Ensure that min dimentions are not set to be bigger than max dimentions,
+  // but only if max dimentions have been specified. This allows to set min
+  // dimentions, without having to restrict max dimentions.
+  if (!empty($max_dimensions_x) && $min_dimensions_x > $max_dimensions_x || !empty($max_dimensions_y) && $min_dimensions_y > $max_dimensions_y) {
+    form_set_error('', t('The minimum image dimentions cannot be bigger than its maximum dimentions.'));
```

Comment:
A red border around a form field would be nice.
Something like `form_set_error('instance][settings][min_dimensions', t('The min....` should do the trick.
While that's not a must, it's helpful for people not knowing, what this form error means.

And - ehem - I'm not a native speaker, but I think it's dimen**s**ions, not dimentions. :wink:

Reply:
Thanks for the suggestion and the catch on that typo @indigoxela 👍 ...fixed!